test/integration/globals/app-nav.js

```diff
@@ -866,6 +958,60 @@ context('App Nav', function() {
     cy.clock().invoke('restore');
   });
 
+  specify('add patient - clinician in one workspace', function() {
+    const testClinician = {
+      id: '1',
+      attributes: {
+        name: 'Test Clinician',
+        email: 'test.clinician@roundingwell.com',
+        enabled: true,
+        last_active_at: testTs(),
+      },
+      relationships: {
+        team: { data: { id: '11111' } },
+        workspaces: { data: [{ id: '11111' }] },
+        role: { data: { id: '22222' } },
+      },
+    };
+
+    cy
+      .routeWorkspaces()
+      .routeSettings(fx => {
+        fx.data.push({ id: 'manual_patient_creation', attributes: { value: true } });
+
+        return fx;
+      })
+      .routeWorkspaceClinicians(fx => {
+        fx.data = _.sample(fx.data, 1);
+        fx.data[0] = testClinician;
+
+        return fx;
+      })
+      .routeCurrentClinician(fx => {
+        fx.data = testClinician;
+
+        return fx;
+      })
+      .routeActions()
+      .routePatient()
+      .visit('/', { isRoot: true });
+
+    cy
+      .get('.app-nav')
+      .find('.js-add-patient')
+      .click();
+
+    cy
+      .get('.modal')
+      .find('[data-workspaces-region] .list-manager__item')
+      .should('contain', 'Workspace One');
+
+    cy
+      .get('.modal')
+      .find('[data-workspaces-region] [data-droplist-region] button')
+      .should('be.disabled');
+  });
+
```

Comment:
This adds coverage for this: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/globals/nav_app.js#L382-L386.